src/cow-react/modules/swap/containers/TradeSummary/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { AdvancedSwapDetailsProps } from 'components/swap/AdvancedSwapDetails'
+
+// Sub-components
+import { useWalletInfo } from 'hooks/useWalletInfo'
+import { useHigherUSDValue } from 'hooks/useStablecoinPrice'
+import { TradeSummaryContent } from '@cow/modules/swap/pure/TradeSummary'
+
+export type TradeSummaryProps = Required<AdvancedSwapDetailsProps> & {
```

Comment:
Let's copy properties from AdvancedSwapDetailsProps to TradeSummaryProps instead of intersection to avoid dependency of `custom` directory from here